# Reports the status of external signals or sensors

bool vision_ok
bool sonar_ok
bool gps_ok
